gif_sizer/parser.py

```diff
@@ -0,0 +1,14 @@
+import click
+
```

Comment:
Some people add docstring at the beginning of each python file. Like that:
```suggestion
"""
Module contains super mega complex method with very complicated 'click' decorators.
"""
```

Reply:
I could agree if this would be for a library development or a script, but in this case it will be a package and I'd expect to have a documentation on class definition level.